vars/kibanaCoverage.groovy

```diff
@@ -125,31 +125,31 @@ def uploadCombinedReports() {
   )
 }
 
-def ingestData(buildNum, buildUrl, title) {
+def ingestData(jobName, buildNum, buildUrl, title) {
   kibanaPipeline.bash("""
     source src/dev/ci_setup/setup_env.sh
     yarn kbn bootstrap --prefer-offline
     # Using existing target/kibana-coverage folder
-    . src/dev/code_coverage/shell_scripts/ingest_coverage.sh ${buildNum} ${buildUrl}
+    . src/dev/code_coverage/shell_scripts/ingest_coverage.sh ${buildNum} ${buildUrl} ${jobName}
```

Comment:
Maybe for consistency it is better put args in same order with `ingestWithVault`:
`${jobName} ${buildNum} ${buildUrl}`

Reply:
Sure, no problem